### PR TITLE
87 tms styling updates

### LINKED ISSUE #87: Styling cleanup shift
Who wants it?

Take out all the %br 's

### PR DESCRIPTION
**Pull request by tmikeschu**
@njgheorghita @Sh1pley @riverswb @ski-climb ready for review.

screen shot 2017-02-12 at 5 54 12 pm

@Sh1pley it looks like that current user method wasn't working in your show view because one of the tests was missing a login stub. I made a before block and stub the login there, and current user works now.

* Extract nav bar logic to partials
* Adjust spacing for nav links
* Remove html breaks from new couch form
* Add height to new form
* Update styling and structure for views  …
* Add default link behavior to skeleton
* Update dashboard view and styling
* Add on-focus selectors to links
* Remove br tags

Closes #87

### CODE REVIEW

File: app/views/users/couches/new.html.haml [hunk 1]
```diff
@@ -1,30 +1,18 @@
...
-      = f.text_field :state
-      %br
-      = f.label :zipcode
-      %br
-      = f.text_field :zipcode
-      %br
-      = f.submit "Add Couch"
\ No newline at end of file
+
+      = f.text_field :name, placeholder: "Name"
```
Comment (riverswb) [inline app/views/users/couches/new.html.haml:6]: Nice!

**Review by riverswb — APPROVED** [review]

**Comment by riverswb** [comment]
I'm assuming the conflicts are pretty much nothing since they are all views stuff

**Comment by tmikeschu** [comment]
Conflicts resolved! @riverswb

File: app/views/layouts/_nav.html.haml [hunk 2]
```diff
@@ -5,10 +5,6 @@
   %section.nav_right
     %ul
       - if current_user
```
Comment (ski-climb) [inline app/views/layouts/_nav.html.haml:7]: partials: ❤️

File: app/views/users/couches/new.html.haml [hunk 3]
```diff
@@ -1,30 +1,18 @@
 %section.vanilla_form
   %h1 Add Couch
-  %br
```
Comment (ski-climb) [inline app/views/users/couches/new.html.haml:3]: 👍 to removing `<br/>`'s

**Review by ski-climb — APPROVED** [review]

**Pull request merged**